source/core/src/main/com/csse3200/game/components/combat/CombatManager.java

```diff
@@ -412,6 +440,64 @@ private void displayCombatResults() {
         }
 
         ServiceLocator.getDialogueBoxService().updateText(moveText);
+
+        // Add the listener to initiate enemy animations when enemy move indicated on dialogue box:
+        addDialogueBoxListener();
+
         entity.getEvents().trigger("displayCombatResults");
     }
+
+    /**
+     * Add listener to continue button in dialogue box in combat to allow syncing of
+     * enemy animations after player animations and when continue button is pressed as enemy attack is
+     * described in dialogue box
+     */
+    public void addDialogueBoxListener() {
+
+        // Get the continue button for the dialogue box
+        contButton = ServiceLocator.getDialogueBoxService().getCurrentOverlay().getForwardButton();
+
+        dialogueBoxCombatListener = new InputListener() {
+            @Override
+            public boolean touchDown(InputEvent event, float x, float y, int pointer, int button) {
+
+                combatAnimationDisplay.dispose();
+
+                Label db = ServiceLocator.getDialogueBoxService().getCurrentOverlay().getLabel();
+                String currentText = String.valueOf(db.getText());
+
+                //            TODO: replace Label code with code below due in next PR
```

Comment:
is this still for the next pr or should this be deleted?

Reply:
yes next pr probably. thanks for reviewing :)